UnityAds/UnityAdsInterstitialCustomEvent+UnityAdsLoadDelegateAdditions.h

```diff
@@ -0,0 +1,36 @@
+//
+//  UnityAdsInterstitial+UnityAdsLoadDelegateAdditions.h
+//  MoPubSDK
+//
+//  Copyright © 2021 MoPub. All rights reserved.
+//
+
+#ifndef UnityAdsInterstitial_UnityAdsLoadDelegateAdditions_h
+#define UnityAdsInterstitial_UnityAdsLoadDelegateAdditions_h
+
+#import <UnityAds/UnityAds.h>
+#import "UnityAdsInterstitialCustomEvent.h"
+
+/**
+ *  The `UnityAdsLoadDelegate` protocol defines the required methods for receiving messages from UnityAds.load() method.
+ */
+@interface UnityAdsInterstitialCustomEvent (UnityAdsLoadDelegate)
+
+/**
+ *  Callback triggered when a load request has successfully filled the specified placementId with an ad that is ready to show.
+ *
+ *  @param placementId The ID of the placement as defined in Unity Ads admin tools.
+ */
+- (void)unityAdsAdLoaded:(NSString * _Nonnull)placementId;
```

Comment:
Im not sure if we want to put those comments here. They might be useful, but we have to maintain them. Correct me if Im wrong but they are copy-paste from our delegate - right?

Reply:
Yeah, I agree with you here.  I was thinking having them here might help reviewers less familiar with our SDK when identifying how things work.  But they could always just look them up.  Maybe instead I will just link to the public repo.